source/_integrations/motionmount.md

```
@@ -17,12 +17,92 @@ ha_platforms:
+  - **Description**: The error status of the MotionMount.
+    - None: There is no error.
+    - Motor: There is a problem communicating with the motor.
+    - Internal: There is an internal error. Refer to the MotionMount app for support.
+
+#### Numbers
+
+- **Extension**
+  - **Description**: The current extension of the MotionMount from the wall.
+- **Turn**
+  - **Description**: The current rotation of the MotionMount.
+     
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 5

(MD009, no-trailing-spaces)